packages/api/src/utils/createTask.ts

```diff
@@ -240,10 +236,17 @@ export const enqueueParseRequest = async (
     return ''
   }
 
+  // use GCF url for low priority tasks
+  const taskHandlerUrl =
```

Comment:
Hard to tell from the diff, would it be cleaner if content fetch also passed the queue name in?

Reply:
Yes, let me change this